Tests/StencilTests/ForNodeSpec.swift

```diff
@@ -111,8 +111,8 @@ func testForNode() {
         try expect(try node.render(context)) == "empty"
     }
 
-    $0.it("can render a filter") {
-      let templateString = "{% for article in ars|default:articles %}" +
+    $0.it("can render a filter with spaces") {
+      let templateString = "{% for article in ars | default: a, b , articles %}" +
```

Comment:
Not sure I'm familiar with that default filter accepting 3 arguments?

Reply:
It supports variadic fallbacks for the default value. If `a` and `b` are not defined then articles would be there.